actions/pypi_build/action.yaml

```diff
@@ -0,0 +1,56 @@
+name: "Build PyPi"
+description: "Build PyPi wheel"
+
+inputs:
+    dev:
+      description: 'If the build is a dev build'
+      required: false
+      default: false
+    release:
+      description: 'If the build is a release build'
+      required: false
+      default: false
+    name:
+      description: 'Name to append to the wheel'
+      required: false
+     
+outputs:
+    filename:
+      description: 'wheel filename'
+      value: ${{ steps.build.outputs.filename }}
+
+# can we just pass in build args instead of updating the variables in version.py?
```

Comment:
yeah, I'd be okay with standardizing maybe some env vars that can override in version.py